src/main/java/net/glowstone/command/minecraft/GameModeCommand.java

```diff
@@ -22,8 +22,7 @@
      * Creates the instance for this command.
      */
     public GameModeCommand() {
-        super("gamemode", "Change the game mode of a player.", "/gamemode <mode> [player]",
-            Collections.emptyList());
+        super("gamemode", "Change the game mode of a player.", "/gamemode <mode> [player]", Collections.emptyList());
```

Comment:
As per our code style, the line length should not exceed 100. Can you revert this change?